docs/guide/controllers.md

```diff
@@ -718,6 +718,12 @@ For example:
 {{ ez_render_field(content, 'posts', {'parameters': {'enablePagination': true, 'itemsPerPage': 8}}) }}
 ```
 
+You can also define an offset for the results. Provide an offset in the Query Type, or in parameters:
+
+```
+offset: 3
```

Comment:
We may have to explain what will happen with the offset:
- if no offset is defined in the query
  - if pagination is disabled, it will use 0
  - if pagination is enabled, it will add the itemsPerPage count to the offset (page1 = 10, page2 = 20, etc)
- if an offset is defined in the query
  - if pagination is disabled, it will use that offset
  - if pagination is disabled, the query's offset will be added to the offset computed for a page (offset = 5, itemsPerPage = 10, page0 = offset 5, page1 = offset 15, page 2 = offset 25)

Reply:
Looks good to me !